examples/client.rs

```diff
@@ -28,7 +29,8 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
         .uri(url)
         .body(Empty::<bytes::Bytes>::new())?;
 
-    let resp = client.request(req).await?;
+    let span = info_span!("request", uri = %req.uri());
+    let resp = client.request(req).instrument(span).await?;
```

Comment:
Could we remove this too?

I agree it's useful when debugging, but I worry it distracts from the initial usage of someone just wanting to get the client working.

Reply:
yep 2bce070..5f7432e